### ISSUE TITLE
verifytime should exit gracefully when versionstamp does not exist

### ISSUE THREAD

**Issue by phmccarty** [clearlinux/swupd-client#289]
The `verifytime` program is a helper program used by swupd-client that will try to correct certain types of time skews, and it relies on the `/usr/share/clear/versionstamp` for a hint about when the current version was released (in seconds since the epoch).

However, when `/usr/share/clear/versionstamp` does not exist, for whatever reason, I think `verifytime` needs to exit gracefully because (a) it lacks required inputs, and (b) `verifytime` failing is not critical to swupd-client's operations. That is, swupd-client's success/failure should not depend on `verifytime`'s success/failure.

**Comment by phmccarty** [comment]
For background, see #234.

**Comment by matthewrsj** [comment]
> I think verifytime needs to exit gracefully

verifytime returns 1 when it fails to verify the system time. This seems reasonable to me. 

Do you mean when `system("verifytime")` fails we need to just continue? What about the comment that states `Check that our system time is reasonably valid before continuing, or the certificate verification will fail with invalid time`?

Users can just pass `-I` to skip this check. Is this not sufficient?

**Comment by phmccarty** [comment]
> verifytime returns 1 when it fails to verify the system time. This seems reasonable to me.

Agreed.

> Do you mean when system("verifytime") fails we need to just continue?

Yes. `verifytime` is trying to address an issue that may prevent signature verification from succeeding. That is its only purpose.

> What about the comment [ ... ]

The comment is valid. If `verifytime` has required inputs, it will try to fix the system time. If not, it will perform no work. Regardless, swupd-client (IMO) should continue and try to verify signatures and should not be gated on an external binary successfully performing work.

The `-I` option can be used if people want to skip signature verification altogether. I would only use that option as a last resort to unblock an update.

**Comment by phmccarty** [comment]
@matthewrsj In other words, I would be in favor of ignoring the exit code from `verifytime` as a solution. That actually sounds better than my original idea.

**Comment by matthewrsj** [comment]
> I would be in favor of ignoring the exit code from verifytime as a solution

Ok this clears it up.

**Comment by matthewrsj** [comment]
Closing in favor of #340
